src/utils/currencyFormatter.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Utility function that converts a currency amount represented in cents to
+ * its string representation in dollars and cents.
+ *
+ * @param amountInCents Amount in cents
+ * @param separator Separator character between dollars and cents
+ * @returns Amount in dollars and cents, as a string
+ */
+export const formatCentsAsDollarsAndCents = (
+  amountInCents: number,
+  separator = "."
+): string => {
+  const amountString = amountInCents.toString();
```

Comment:
Can be simplified with a small change.
`return (amountInCents / 100).toLocaleString("en-US", { minimumFractionDigits: 2, });`.
The test you've written should still pass. 😁

Reply:
This is done in [1aff2ff](https://github.com/rationally-app/mobile-application/pull/609/commits/1aff2ff755e0ba3455c50ce1fa13c1a01f527891)!